ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -349,7 +349,7 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
                                              target: nil,
                                              action: #selector(NavigationBarBackButtonDelegate.backButtonWasPressed))
         backButtonItem.accessibilityIdentifier = "Back"
-        backButtonItem.accessibilityLabel = "Accessibility.NavigationBar.BackLabel".localized
+        backButtonItem.accessibilityLabel = StringKeys.accessibilityNavigationBarBackLabel.localized
```

Comment:
I don't think this change is necessary especially if the new class is created temporarily. We should limit its usage to be strictly external and not internal so we don't have to touch this again when StringKeys changes later on.

Reply:
for code maintainability though, someone could change internally and break the external usage without knowing it?

Reply:
I doubt this would change but even if a change were to happen it would probably be the value of the string and not the key itself that would change and that shouldn't result in any breaks as far as I can tell.

Reply:
yeah oops, ill make a change to expose the localized string